site/content/contribute/mobile/developer-setup.md

```diff
@@ -210,3 +210,22 @@ xcrun: error: unable to find utility "instruments", not a developer tool or in P
   ![](/img/xcode_preferences.png)
 
   - After this go back to the command line and run ``make run-ios`` again.
+
+### Errors When Running react-native packager
```

Comment:
Propose `Errors When Running 'react-native packager'` for consistency with previous section